terra-workspace-manager-jmeter/README.md

```diff
@@ -0,0 +1,33 @@
+# terra-workspace-manager-jmeter
```

Comment:
Looks like this file was missed when updating the language from `master/slave` to `director/worker`

Reply:
Renamed everything to director/worker language.